app/src/main/res/values/strings.xml

```diff
@@ -26,154 +26,154 @@
+    <string name="title_ssh_host_key_changed">The host key changed!</string>
     <string name="title_continue">Continue</string>
-    <string name="title_delete_servers_before_import">Delete current Servers before Import</string>
+    <string name="title_delete_servers_before_import">Remove all servers before importing</string>
     <string name="title_password">Password</string>
-    <string name="title_use_repeater">Use Repeater</string>
-    <string name="title_private_key">Private Key</string>
+    <string name="title_use_repeater">Repeater</string>
+    <string name="title_private_key">Private key</string>
     <string name="title_change">Change</string>
     <string name="title_about">About</string>
-    <string name="title_license">Licence</string>
-    <string name="title_open_source_libraries">Open Source Libraries</string>
-    <string name="title_report_bug">Report a Bug</string>
-    <string name="title_github_repo">Github Repo</string>
+    <string name="title_license">GPLv3+ licence</string>
+    <string name="title_open_source_libraries">Libre libraries</string>
+    <string name="title_report_bug">Report</string>
+    <string name="title_github_repo">Code</string>
 
-    <string name="hint_urlbar">Connect to a Server</string>
+    <string name="hint_urlbar">Server URL…</string>
     <string name="hint_server_name">Name (e.g. Work PC)</string>
     <string name="hint_host">Host</string>
-    <string name="hint_repeater_host">Repeater Host</string>
+    <string name="hint_repeater_host">Repeater host</string>
     <string name="hint_port">Port</string>
     <string name="hint_server_id">Server ID</string>
     <string name="hint_username">Username</string>
     <string name="hint_password">Password</string>
-    <string name="hint_key_password">Key Password</string>
+    <string name="hint_key_password">Key password</string>
 
-    <string name="msg_about_app">AVNC is an open-source VNC client for Android. It allows you to remotely control any device running a VNC server.</string>
-    <string name="msg_server_profile_added">Server added.</string>
-    <string name="msg_server_profile_deleted">Server deleted.</string>
-    <string name="msg_pip_not_supported">Picture-in-Picture is not supported on this device</string>
-    <string name="msg_ptr_hiding_not_supported">Hiding cursor is not supported on this device</string>
-    <string name="msg_loading">Loading …</string>
-    <string name="msg_imported">Imported Successfully</string>
-    <string name="msg_exported">Exported Successfully</string>
-    <string name="msg_export_auth_required">Authentication is required before Export.</string>
-    <string name="msg_export_auth_missing">Your device is not secured. Turn on screen lock to enable authentication before Export.</string>
-    <string name="msg_invalid_key_file">File is not a valid private key!</string>
+    <string name="msg_about_app">A copylefted libre VNC client.\nUse any device with a VNC server remotely.</string>
```

Comment:
```suggestion
    <string name="msg_about_app">A copylefted libre VNC client.\nRemotely use any device running a VNC server.</string>
```
I think we should also keep the word `open-source`. While I understand the motivation behind `Libre`, we are talking about user-facing strings here, and user are more likely to understand what `open-source` means. We can use whatever we want in project's Readme file.